mono/mini/Makefile.am.in

```diff
@@ -724,26 +747,40 @@ gsharedvtcheck:
 
 fullaot_regtests = $(regtests) aot-tests.exe $(if $(GSHAREDVT),gshared.exe)
 
-FULLAOT_LIBS = \
-	mscorlib.dll \
-	System.Core.dll \
-	System.dll \
+FULLAOT_LIBS_MOBILE_STATIC_BLACKLIST = \
```

Comment:
Why are those libs being blacklisted? Please add it as a comment so we can act on it over time.